bugswarm/client/docker.py

```diff
@@ -3,13 +3,18 @@
 import sys
 
 from bugswarm.common import log
-from bugswarm.common.credentials import DOCKER_HUB_REPO, DOCKER_HUB_CACHED_REPO
 from bugswarm.common.shell_wrapper import ShellWrapper
+import bugswarm.common.credentials as credentials
 
 SCRIPT_DEFAULT = '/bin/bash'
 HOST_SANDBOX_DEFAULT = '~/bugswarm-sandbox'
 CONTAINER_SANDBOX_DEFAULT = '/bugswarm-sandbox'
 
+DOCKER_HUB_REPO = \
+    credentials.DOCKER_HUB_REPO if hasattr(credentials, 'DOCKER_HUB_REPO') else 'bugswarm/images'
+DOCKER_HUB_CACHED_REPO = \
+    credentials.DOCKER_HUB_CACHED_REPO if hasattr(credentials, 'DOCKER_HUB_CACHED_REPO') else 'bugswarm/cached-images'
```

Comment:
Not sure these will work. By default in `credentials.py`, `DOCKER_HUB_REPO = ''` and `DOCKER_HUB_CACHED_REPO = ''`. So `hasattr(credentials, 'DOCKER_HUB_REPO')` and `hasattr(credentials, 'DOCKER_HUB_CACHED_REPO')` will always return `True`.

Reply:
Oh! I see.